csm/conf/configure.py

```diff
@@ -153,6 +156,7 @@ def _set_csm_endpoint():
 
     @staticmethod
     def _set_s3_endpoints():
+        Log.info("Setting S3 endpoints in csm config")
```

Comment:
Setting S3 endpoints in csm config -> Setting S3 endpoints in configuration. 
Be consistent with the words you are choosing. Like config -> configuration